scripts/setup-auth-qa.sh

```diff
@@ -13,8 +13,15 @@ fi
 
 QA="${PROJECT_NAME}QA"
 
+read -p "Enter login domain [test.salesforce.com]" LOGIN_DOMAIN
```

Comment:
Looking good, I like that the prompt adopts familiar formatting I see when e.g. I do an `npm init` and work thru those prompts 👍 
I'd suggest throwing a colon in there ala:
```
Enter login domain: (test.salesforce.com)
```

Reply:
Yeah, I was going off what I remembered from the AWS CLI configure. 

What would be the process for making that change? Would I:
1. cancel this pull request
1. make a new commit
1. make a new pull request